`ENABLE_BRAIN=0` causes compile error ("fatal error: xxhash.h: No such file or directory")

**Describe the bug**

hashcat does not compile with `ENABLE_BRAIN` set to 0.

**To Reproduce**

1. Clone latest master
2. Change `ENABLE_BRAIN` to 0 in `src/Makefile`
3. `make -j4`

```make
gcc    -std=gnu99 -march=native -W -Wall -Wextra -O2 -pipe -Iinclude/ -IOpenCL/ -Ideps/LZMA-SDK/C -Ideps/zlib -Ideps/zlib/contrib -Ideps/OpenCL-Headers -DWITH_CUBIN -Ideps/unrar -DWITH_HWMON src/feeds/feed_wordlist.c obj/combined.NATIVE.a -o feeds/feed_wordlist.so -s -lstdc++ -lpthread -ldl -lrt -lm -shared -fPIC -D FEEDS_INTERFACE_VERSION_CURRENT=713
src/feeds/feed_wordlist.c:15:10: fatal error: xxhash.h: No such file or directory
   15 | #include "xxhash.h"
      |          ^~~~~~~~~~
compilation terminated.
make: *** [src/Makefile:880: feeds/feed_wordlist.so] Error 1
make: *** Waiting for unfinished jobs....
```

When I `make clean` and undo the `ENABLE_BRAIN` change, hashcat compiles.

--- a/src/Makefile
+++ b/src/Makefile
@@ -324,13 +324,15 @@ endif
 # OpenCL
 CFLAGS                  += -I$(DEPS_OPENCL_PATH)
 
-# brain and xxHash
+# brain
 ifeq ($(ENABLE_BRAIN),1)
 CFLAGS                  += -DWITH_BRAIN
+endif
+
+# xxHash
 CFLAGS                  += -I$(DEPS_XXHASH_PATH)
 ifeq ($(USE_SYSTEM_XXHASH),1)
 LFLAGS                  += -lxxhash
-endif
 endif
 
 # CUDA binary cache

